src/templates/dandelion/config/screens/LockScreen.js

```diff
@@ -0,0 +1,338 @@
+      setFormError(error)
+
+      if (!error) {
+        const screenData = {
+          lockToken: lockToken,
+          lockDuration,
+          spamPenalty,
+          lockAmount,
+        }
+        next(dataKey ? { ...data, [dataKey]: screenData } : screenData)
+      }
+    },
+    [lockDuration, lockToken, lockAmount, spamPenalty, next, dataKey, data]
+  )
+
+  return (
+    <form
+      css={`
+        display: grid;
+        align-items: center;
+        justify-content: center;
+      `}
+    >
+      <Header
+        title={title}
+        subtitle={
+          <span
+            css={`
+              display: flex;
+              align-items: center;
+              justify-content: center;
+            `}
+          >
+            Choose your
+            <span
+              css={`
+                display: flex;
+                margin: 0 ${1.5 * GU}px;
+              `}
+            >
+              <KnownAppBadge
+                appName="time-lock.aragonpm.eth"
+                label={appLabel}
+              />
+            </span>
+            settings below.
+          </span>
+        }
+      />
+
+      <div
+        css={`
+          ${fieldsLayout};
+        `}
+      >
+        <Field
+          label={
+            <React.Fragment>
+              Token address
+              <Help hint="What’s the token address?">
+                <strong>Token address</strong> is the address of the token to be
+                locked.
+              </Help>
+            </React.Fragment>
+          }
+        >
+          <TokenSelector
+            selectedIndex={lockToken.selectedIndex}
+            onChange={handleTokenChange}
+            value={lockToken.data.address}
+            tokens={[DEFAULT_LOCK_TOKEN]}
+          />
+        </Field>
+
+        <Field
+          label={
+            <React.Fragment>
+              Lock amount
+              <Help hint="What’s the lock amount?">
+                <strong>Lock amount</strong> is the amount of tokens to lock.
+              </Help>
+            </React.Fragment>
+          }
+        >
+          <TextInput
+            onChange={handleLockAmountChange}
+            value={lockAmount === -1 ? '' : lockAmount}
+            wide
+          />
+        </Field>
+      </div>
+
+      <Duration
+        duration={lockDuration}
+        onUpdate={handleDurationChange}
+        label={
+          <React.Fragment>
+            {DURATION_LABEL}
+            <Help hint="What’s the lock duration?">{DURATION_HELP_TEXT}</Help>
+          </React.Fragment>
+        }
+      />
+
+      <Field
+        label={
+          <React.Fragment>
+            Spam penalty %
+            <Help hint="What’s the spam penalty?">
+              <strong>Spam penalty</strong>
+              Is a % of the base lock amount and lock duration. The more active
+              locks an account has, the more tokens will have to lock and for a
+              longer period of time for subsequent proposals.
```

Comment:
I'm not sure I really understand this; could we rewrite this copy?

Reply:
How about:

> For each additional active lock a user creates, the required amount of tokens and duration of the next lock will be increased by this percentage of the base amount and duration.

Reply:
SGTM

Reply:
Yes, you can see the formula here https://github.com/1Hive/time-lock-app/blob/639c696a0b6d561b70481539ec1a2cc55ea2f546/contracts/TimeLock.sol#L200